### ISSUE TITLE
Multiple custom List matchers fail in 1.10.2

### ISSUE THREAD

**Issue by dzirbel** [mockk/mockk#503]
MockK 1.10.2 seems to have introduced a strange behavior when mocking and matching functions with `List` parameters. From the error message, it appears that custom matchers beyond the first are being replaced with `eq([])`.

I've omitted most of the sections here; the minimal code block before describes the behavior best. This test ought to pass (and does in 1.10.0), but fails with the error message below in 1.10.2.

Some notes:
- the test passes with only one verify call, it seems that having multiple matchers is related to the issue
- it also seems to be related to `List` being the parameter type - if the function argument is a `String`, the matchers appear to work as expected

#### Context

* MockK version: 1.10.2 (confirmed to pass in 1.10.0; have not tested in 1.10.1)
* OS: Linux
* Kotlin version: 1.4.10
* JDK version: 1.8
* JUnit version: 5.7.0
* Type of test: unit test

#### Minimal reproducible code (the gist of this issue)

```
import io.mockk.every
import io.mockk.mockk
import io.mockk.verify
import org.junit.jupiter.api.Test

internal class MockkTest {
    class TestObject {
        fun foo(list: List<String>): Boolean = list.isEmpty()
    }

    @Test
    fun test() {
        val mock = mockk<TestObject> {
            every { foo(any()) } returns false
        }

        mock.foo(listOf("1", "2"))
        mock.foo(listOf("3"))

        verify {
            mock.foo(match { it.size == 2 })
            mock.foo(match { it.size == 1 })
        }
    }
}
```

#### Stack trace

```
java.lang.AssertionError: Verification failed: call 2 of 2: TestObject(#1).foo(eq([]))). No matching calls found.

Calls to same method:
1) TestObject(#1).foo([1, 2])
2) TestObject(#1).foo([3])


Stack traces:
1)                                                           io.mockk.impl.InternalPlatform.captureStackTrace           (InternalPlatform.kt:121)                          
                                                               io.mockk.impl.stub.MockKStub.handleInvocation            (MockKStub.kt:247)                                 
                             io.mockk.impl.instantiation.JvmMockFactoryHelper$mockHandler$1.invocation                  (JvmMockFactoryHelper.kt:25)                       
                                                      io.mockk.proxy.jvm.advice.Interceptor.call                        (Interceptor.kt:20)                                
                                               my.package.MockkTest$TestObject.foo                         (MockkTest.kt:10)                                  
                                                          my.package.MockkTest.test                        (MockkTest.kt:19)                                  
                                                       sun.reflect.NativeMethodAccessorImpl.invoke0                     (NativeMethodAccessorImpl.java:-2)N                
                                                       sun.reflect.NativeMethodAccessorImpl.invoke                      (NativeMethodAccessorImpl.java:62)                 
                                                   sun.reflect.DelegatingMethodAccessorImpl.invoke                      (DelegatingMethodAccessorImpl.java:43)             
                                                                   java.lang.reflect.Method.invoke                      (Method.java:498)                                  
                                            org.junit.platform.commons.util.ReflectionUtils.invokeMethod                (ReflectionUtils.java:688)                         
                                        org.junit.jupiter.engine.execution.MethodInvocation.proceed                     (MethodInvocation.java:60)                         
         org.junit.jupiter.engine.execution.InvocationInterceptorChain$ValidatingInvocation.proceed                     (InvocationInterceptorChain.java:131)              
                                        org.junit.jupiter.engine.extension.TimeoutExtension.intercept                   (TimeoutExtension.java:149)                        
                                        org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestableMethod     (TimeoutExtension.java:140)                        
                                        org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestMethod         (TimeoutExtension.java:84)                         
             org.junit.jupiter.engine.execution.ExecutableInvoker$ReflectiveInterceptorCall.lambda$ofVoidMethod$0       (ExecutableInvoker.java:115)                       
                                       org.junit.jupiter.engine.execution.ExecutableInvoker.lambda$invoke$0             (ExecutableInvoker.java:105)                       
        org.junit.jupiter.engine.execution.InvocationInterceptorChain$InterceptedInvocation.proceed                     (InvocationInterceptorChain.java:106)              
                              org.junit.jupiter.engine.execution.InvocationInterceptorChain.proceed                     (InvocationInterceptorChain.java:64)               
                              org.junit.jupiter.engine.execution.InvocationInterceptorChain.chainAndInvoke              (InvocationInterceptorChain.java:45)               
                              org.junit.jupiter.engine.execution.InvocationInterceptorChain.invoke                      (InvocationInterceptorChain.java:37)               
                                       org.junit.jupiter.engine.execution.ExecutableInvoker.invoke                      (ExecutableInvoker.java:104)                       
                                       org.junit.jupiter.engine.execution.ExecutableInvoker.invoke                      (ExecutableInvoker.java:98)                        
                               org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeTestMethod$6   (TestMethodTestDescriptor.java:210)                
                          org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute                     (ThrowableCollector.java:73)                       
                               org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeTestMethod            (TestMethodTestDescriptor.java:206)                
                               org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute                     (TestMethodTestDescriptor.java:131)                
                               org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute                     (TestMethodTestDescriptor.java:65)                 
                                org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5 (NodeTestTask.java:139)                            
                          org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute                     (ThrowableCollector.java:73)                       
                                org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7 (NodeTestTask.java:129)                            
                                        org.junit.platform.engine.support.hierarchical.Node.around                      (Node.java:137)                                    
                                org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8 (NodeTestTask.java:127)                            
                          org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute                     (ThrowableCollector.java:73)                       
                                org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively          (NodeTestTask.java:126)                            
                                org.junit.platform.engine.support.hierarchical.NodeTestTask.execute                     (NodeTestTask.java:84)                             
                                                                        java.util.ArrayList.forEach                     (ArrayList.java:1257)                              
   org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll                   (SameThreadHierarchicalTestExecutorService.java:38)
                                org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5 (NodeTestTask.java:143)                            
                          org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute                     (ThrowableCollector.java:73)                       
                                org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7 (NodeTestTask.java:129)                            
                                        org.junit.platform.engine.support.hierarchical.Node.around                      (Node.java:137)                                    
                                org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8 (NodeTestTask.java:127)                            
                          org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute                     (ThrowableCollector.java:73)                       
                                org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively          (NodeTestTask.java:126)                            
                                org.junit.platform.engine.support.hierarchical.NodeTestTask.execute                     (NodeTestTask.java:84)                             
                                                                        java.util.ArrayList.forEach                     (ArrayList.java:1257)                              
   org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll                   (SameThreadHierarchicalTestExecutorService.java:38)
                                org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5 (NodeTestTask.java:143)                            
                          org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute                     (ThrowableCollector.java:73)                       
                                org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7 (NodeTestTask.java:129)                            
                                        org.junit.platform.engine.support.hierarchical.Node.around                      (Node.java:137)                                    
                                org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8 (NodeTestTask.java:127)                            
                          org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute                     (ThrowableCollector.java:73)                       
                                org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively          (NodeTestTask.java:126)                            
                                org.junit.platform.engine.support.hierarchical.NodeTestTask.execute                     (NodeTestTask.java:84)                             
   org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit                      (SameThreadHierarchicalTestExecutorService.java:32)
                    org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute                     (HierarchicalTestExecutor.java:57)                 
                      org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute                     (HierarchicalTestEngine.java:51)                   
                               org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute                     (EngineExecutionOrchestrator.java:108)             
                               org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute                     (EngineExecutionOrchestrator.java:88)              
                               org.junit.platform.launcher.core.EngineExecutionOrchestrator.lambda$execute$0            (EngineExecutionOrchestrator.java:54)              
                               org.junit.platform.launcher.core.EngineExecutionOrchestrator.withInterceptedStreams      (EngineExecutionOrchestrator.java:67)              
                               org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute                     (EngineExecutionOrchestrator.java:52)              
                                           org.junit.platform.launcher.core.DefaultLauncher.execute                     (DefaultLauncher.java:96)                          
                                           org.junit.platform.launcher.core.DefaultLauncher.execute                     (DefaultLauncher.java:75)                          
                                                   com.intellij.junit5.JUnit5IdeaTestRunner.startRunnerWithArgs         (JUnit5IdeaTestRunner.java:69)                     
                                              com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs         (IdeaTestRunner.java:33)                           
                                                         com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart      (JUnitStarter.java:230)                            
                                                         com.intellij.rt.junit.JUnitStarter.main                        (JUnitStarter.java:58)                             

2)                                                           io.mockk.impl.InternalPlatform.captureStackTrace           (InternalPlatform.kt:121)                          
                                                               io.mockk.impl.stub.MockKStub.handleInvocation            (MockKStub.kt:247)                                 
                             io.mockk.impl.instantiation.JvmMockFactoryHelper$mockHandler$1.invocation                  (JvmMockFactoryHelper.kt:25)                       
                                                      io.mockk.proxy.jvm.advice.Interceptor.call                        (Interceptor.kt:20)                                
                                               my.package.MockkTest$TestObject.foo                         (MockkTest.kt:10)                                  
                                                          my.package.MockkTest.test                        (MockkTest.kt:20)                                  
                                                       sun.reflect.NativeMethodAccessorImpl.invoke0                     (NativeMethodAccessorImpl.java:-2)N                
                                                       sun.reflect.NativeMethodAccessorImpl.invoke                      (NativeMethodAccessorImpl.java:62)                 
                                                   sun.reflect.DelegatingMethodAccessorImpl.invoke                      (DelegatingMethodAccessorImpl.java:43)             
                                                                   java.lang.reflect.Method.invoke                      (Method.java:498)                                  
                                            org.junit.platform.commons.util.ReflectionUtils.invokeMethod                (ReflectionUtils.java:688)                         
                                        org.junit.jupiter.engine.execution.MethodInvocation.proceed                     (MethodInvocation.java:60)                         
         org.junit.jupiter.engine.execution.InvocationInterceptorChain$ValidatingInvocation.proceed                     (InvocationInterceptorChain.java:131)              
                                        org.junit.jupiter.engine.extension.TimeoutExtension.intercept                   (TimeoutExtension.java:149)                        
                                        org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestableMethod     (TimeoutExtension.java:140)                        
                                        org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestMethod         (TimeoutExtension.java:84)                         
             org.junit.jupiter.engine.execution.ExecutableInvoker$ReflectiveInterceptorCall.lambda$ofVoidMethod$0       (ExecutableInvoker.java:115)                       
                                       org.junit.jupiter.engine.execution.ExecutableInvoker.lambda$invoke$0             (ExecutableInvoker.java:105)                       
        org.junit.jupiter.engine.execution.InvocationInterceptorChain$InterceptedInvocation.proceed                     (InvocationInterceptorChain.java:106)              
                              org.junit.jupiter.engine.execution.InvocationInterceptorChain.proceed                     (InvocationInterceptorChain.java:64)               
                              org.junit.jupiter.engine.execution.InvocationInterceptorChain.chainAndInvoke              (InvocationInterceptorChain.java:45)               
                              org.junit.jupiter.engine.execution.InvocationInterceptorChain.invoke                      (InvocationInterceptorChain.java:37)               
                                       org.junit.jupiter.engine.execution.ExecutableInvoker.invoke                      (ExecutableInvoker.java:104)                       
                                       org.junit.jupiter.engine.execution.ExecutableInvoker.invoke                      (ExecutableInvoker.java:98)                        
                               org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeTestMethod$6   (TestMethodTestDescriptor.java:210)                
                          org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute                     (ThrowableCollector.java:73)                       
                               org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeTestMethod            (TestMethodTestDescriptor.java:206)                
                               org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute                     (TestMethodTestDescriptor.java:131)                
                               org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute                     (TestMethodTestDescriptor.java:65)                 
                                org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5 (NodeTestTask.java:139)                            
                          org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute                     (ThrowableCollector.java:73)                       
                                org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7 (NodeTestTask.java:129)                            
                                        org.junit.platform.engine.support.hierarchical.Node.around                      (Node.java:137)                                    
                                org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8 (NodeTestTask.java:127)                            
                          org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute                     (ThrowableCollector.java:73)                       
                                org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively          (NodeTestTask.java:126)                            
                                org.junit.platform.engine.support.hierarchical.NodeTestTask.execute                     (NodeTestTask.java:84)                             
                                                                        java.util.ArrayList.forEach                     (ArrayList.java:1257)                              
   org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll                   (SameThreadHierarchicalTestExecutorService.java:38)
                                org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5 (NodeTestTask.java:143)                            
                          org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute                     (ThrowableCollector.java:73)                       
                                org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7 (NodeTestTask.java:129)                            
                                        org.junit.platform.engine.support.hierarchical.Node.around                      (Node.java:137)                                    
                                org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8 (NodeTestTask.java:127)                            
                          org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute                     (ThrowableCollector.java:73)                       
                                org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively          (NodeTestTask.java:126)                            
                                org.junit.platform.engine.support.hierarchical.NodeTestTask.execute                     (NodeTestTask.java:84)                             
                                                                        java.util.ArrayList.forEach                     (ArrayList.java:1257)                              
   org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll                   (SameThreadHierarchicalTestExecutorService.java:38)
                                org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5 (NodeTestTask.java:143)                            
                          org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute                     (ThrowableCollector.java:73)                       
                                org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7 (NodeTestTask.java:129)                            
                                        org.junit.platform.engine.support.hierarchical.Node.around                      (Node.java:137)                                    
                                org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8 (NodeTestTask.java:127)                            
                          org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute                     (ThrowableCollector.java:73)                       
                                org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively          (NodeTestTask.java:126)                            
                                org.junit.platform.engine.support.hierarchical.NodeTestTask.execute                     (NodeTestTask.java:84)                             
   org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit                      (SameThreadHierarchicalTestExecutorService.java:32)
                    org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute                     (HierarchicalTestExecutor.java:57)                 
                      org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute                     (HierarchicalTestEngine.java:51)                   
                               org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute                     (EngineExecutionOrchestrator.java:108)             
                               org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute                     (EngineExecutionOrchestrator.java:88)              
                               org.junit.platform.launcher.core.EngineExecutionOrchestrator.lambda$execute$0            (EngineExecutionOrchestrator.java:54)              
                               org.junit.platform.launcher.core.EngineExecutionOrchestrator.withInterceptedStreams      (EngineExecutionOrchestrator.java:67)              
                               org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute                     (EngineExecutionOrchestrator.java:52)              
                                           org.junit.platform.launcher.core.DefaultLauncher.execute                     (DefaultLauncher.java:96)                          
                                           org.junit.platform.launcher.core.DefaultLauncher.execute                     (DefaultLauncher.java:75)                          
                                                   com.intellij.junit5.JUnit5IdeaTestRunner.startRunnerWithArgs         (JUnit5IdeaTestRunner.java:69)                     
                                              com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs         (IdeaTestRunner.java:33)                           
                                                         com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart      (JUnitStarter.java:230)                            
                                                         com.intellij.rt.junit.JUnitStarter.main                        (JUnitStarter.java:58)                             

	at io.mockk.impl.recording.states.VerifyingState.failIfNotPassed(VerifyingState.kt:66)
	at io.mockk.impl.recording.states.VerifyingState.recordingDone(VerifyingState.kt:42)
	at io.mockk.impl.recording.CommonCallRecorder.done(CommonCallRecorder.kt:47)
	at io.mockk.impl.eval.RecordedBlockEvaluator.record(RecordedBlockEvaluator.kt:60)
	at io.mockk.impl.eval.VerifyBlockEvaluator.verify(VerifyBlockEvaluator.kt:30)
	at io.mockk.MockKDsl.internalVerify(API.kt:118)
	at io.mockk.MockKKt.verify(MockK.kt:149)
	at io.mockk.MockKKt.verify$default(MockK.kt:146)
	at my.package.MockkTest.test(MockkTest.kt:22)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.platform.commons.util.ReflectionUtils.invokeMethod(ReflectionUtils.java:688)
	at org.junit.jupiter.engine.execution.MethodInvocation.proceed(MethodInvocation.java:60)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$ValidatingInvocation.proceed(InvocationInterceptorChain.java:131)
	at org.junit.jupiter.engine.extension.TimeoutExtension.intercept(TimeoutExtension.java:149)
	at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestableMethod(TimeoutExtension.java:140)
	at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestMethod(TimeoutExtension.java:84)
	at org.junit.jupiter.engine.execution.ExecutableInvoker$ReflectiveInterceptorCall.lambda$ofVoidMethod$0(ExecutableInvoker.java:115)
	at org.junit.jupiter.engine.execution.ExecutableInvoker.lambda$invoke$0(ExecutableInvoker.java:105)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$InterceptedInvocation.proceed(InvocationInterceptorChain.java:106)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.proceed(InvocationInterceptorChain.java:64)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.chainAndInvoke(InvocationInterceptorChain.java:45)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.invoke(InvocationInterceptorChain.java:37)
	at org.junit.jupiter.engine.execution.ExecutableInvoker.invoke(ExecutableInvoker.java:104)
	at org.junit.jupiter.engine.execution.ExecutableInvoker.invoke(ExecutableInvoker.java:98)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeTestMethod$6(TestMethodTestDescriptor.java:210)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeTestMethod(TestMethodTestDescriptor.java:206)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:131)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:65)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7(NodeTestTask.java:129)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:127)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:126)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:84)
	at java.util.ArrayList.forEach(ArrayList.java:1257)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:38)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:143)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7(NodeTestTask.java:129)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:127)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:126)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:84)
	at java.util.ArrayList.forEach(ArrayList.java:1257)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:38)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:143)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7(NodeTestTask.java:129)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:127)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:126)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:84)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:32)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:51)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:108)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:88)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.lambda$execute$0(EngineExecutionOrchestrator.java:54)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.withInterceptedStreams(EngineExecutionOrchestrator.java:67)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:52)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:96)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:75)
	at com.intellij.junit5.JUnit5IdeaTestRunner.startRunnerWithArgs(JUnit5IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:33)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:230)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:58)
```

**Comment by jbebar** [comment]
Hello, I have similar issue when trying to match two arguments of type list https://github.com/mockk/mockk/issues/510#issue-720766459

**Comment by Raibaz** [comment]
This was fixed in 1.10.3.
